pubspec.yaml

```diff
@@ -0,0 +1,9 @@
+name: many_points
+version: 0.0.1
+description: A command-line library for visualising data matrices.
+environment:
+  sdk: '>=1.0.0 <2.0.0'
+dependencies:
+  image: '>=1.0.0 <2.0.0'
```

Comment:
This is too broad, and means that the image library can break your code at a later date. Take a look at what version is being pulled in by looking at the lock file, and adjust the style so it's like the test dependency below. I.e. you need to freeze the second most significant version number not the most significant one.